sql/moz-fx-data-shared-prod/firefox_accounts_derived/funnel_events_source_v1/metadata.yaml

```diff
@@ -11,7 +11,6 @@ owners:
 labels: {}
 scheduling:
   dag_name: bqetl_event_rollup
-  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
```

Comment:
Is this an intended change here?

Reply:
This is intended the source query defines all the columns being read